superbench/benchmarks/micro_benchmarks/ib_validation_performance.py

```diff
@@ -43,6 +44,13 @@ def add_parser_arguments(self):
             required=False,
             help='The IB device, e.g., mlx5_0, mlx5_$LOCAL_RANK, mlx5_$((LOCAL_RANK/2)), etc.',
         )
+        self._parser.add_argument(
+            '--set_ib_devices',
+            action='store_true',
+            default=False,
+            help='Set irregular IB devices automatically according to the local rank. \
+            If IB devices are not able to be probed, use env IB_DEVICES to set them manually.',
```

Comment:
if you set an env, it can be used in `--ib_dev` directly? e.g., `mlx5_$(echo "$IB_DEVICES" | cut -d, -f$((LOCAL_RANK + 1)))`